src/main/java/org/trypticon/pdn/paintdotnet/BitmapLayer.java

```diff
@@ -66,8 +67,8 @@ public BitmapLayerProperties(NrbfClassRecord record) {
                     blendOp = new UserBlendOps.XorBlendOp(blendOpRecord);
                     break;
                 default:
-                    throw new UnsupportedOperationException("Unsupported blend op: " +
-                            blendOpRecord.getMetadata().getName());
+                    blendOp = new UserBlendOps.NormalBlendOp(blendOpRecord);
+                    break;

```

Comment:
I left this one out - supporting the blend modes properly is a TODO though. I have a local change attempting to implement Multiply but it is really hard to get the blending right. :(

Depending on which blend mode you encountered though, it might be easier to support whichever one you encountered.